### DISCUSSION TITLE
Replace screenshots with a banner

### DISCUSSION THREAD

**Discussion by KrishnaSSH** [breezy-weather/breezy-weather#1196] [Ideas]
### Describe your idea

Hello @papjul, I have a Suggestion for the Project
An Application mockup themed according to the Project Instead of using raw screenshots of the application UI would be great.

### Other details
here is a Mockup that I made in Figma.
![mockup](https://github.com/user-attachments/assets/f60ac4fa-3bd1-4e48-b4db-d364dd38dda9)
Tell me if there are improvements needed :)

### Acknowledgements

- [X] I have searched the [existing issues](https://github.com/breezy-weather/breezy-weather/issues) and [existing discussions](https://github.com/breezy-weather/breezy-weather/discussions/categories/ideas) and this is a new ticket, **NOT** a duplicate or related to another open OR closed issue or discussion.
- [X] The idea I’m suggesting is not in the [list of features that will not be implemented](https://github.com/breezy-weather/breezy-weather/blob/main/README.md#features-that-will-not-be-implemented).
- [X] I have updated the app to **[version 5.2.5 or later](https://github.com/breezy-weather/breezy-weather/releases/latest)**. Even if I’m getting updates from F-Droid default repo, I don’t need to uninstall/reinstall to update.
- [X] I wrote my request in English, and any screenshot I attached are also in English (language can be changed from Settings > Appearance). I can write in my native language under the English text, if necessary.
- [X] I understand that this app is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY, so I may not get a positive reply to my request, if any at all.

**Comment by papjul** [comment]
Hello, please fill in the template

**Reply by KrishnaSSH** [reply]
> Hello, please fill in the template

how?

**Reply by papjul** [reply]
Here:
![Screenshot_20240714_122706](https://github.com/user-attachments/assets/efb3267b-4de5-49a2-a1ab-d79ad7064111)

Unfortunately, GitHub doesn't allow you to access the original form, so I'm also joining a screenshot of the original form that is more detailed if that helps:
![Screenshot_20240714_122839](https://github.com/user-attachments/assets/843bf6bc-5bf9-4b60-a1e2-37f3970e9500)

**Reply by KrishnaSSH** [reply]
> Here: ![Screenshot_20240714_122706](https://private-user-images.githubusercontent.com/1449159/348535616-efb3267b-4de5-49a2-a1ab-d79ad7064111.jpeg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjA5NTc2MTcsIm5iZiI6MTcyMDk1NzMxNywicGF0aCI6Ii8xNDQ5MTU5LzM0ODUzNTYxNi1lZmIzMjY3Yi00ZGU1LTQ5YTItYTFhYi1kNzlhZDcwNjQxMTEuanBlZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MTQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzE0VDExNDE1N1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTU1YzdjMjZmZGI2MGRkZWY5NGI0NTU4NjJlZTg3M2ZjODE3OGY3ODU3NTQ3OWM4YWM5NjZhMmExMWYwYzZiNTkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.wrYa8KS3w1Z-zP07VphkCNj_TdkgsWrrsVWDNmQ93-E)
> 
> Unfortunately, GitHub doesn't allow you to access the original form, so I'm also joining a screenshot of the original form that is more detailed if that helps: ![Screenshot_20240714_122839](https://private-user-images.githubusercontent.com/1449159/348535724-843bf6bc-5bf9-4b60-a1e2-37f3970e9500.jpeg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjA5NTc2MTcsIm5iZiI6MTcyMDk1NzMxNywicGF0aCI6Ii8xNDQ5MTU5LzM0ODUzNTcyNC04NDNiZjZiYy01YmY5LTRiNjAtYTFlMi0zN2YzOTcwZTk1MDAuanBlZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MTQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzE0VDExNDE1N1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQzY2JiNTBjZmE3OTg4MTIzMTY2OTk5ZWE2YTVkNThjYzU1MTU0ZDA0Yzc1MDUwMTJlY2JlZDhmMjczYjkxYjAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.hEgqtYgYjQ-H0a4pI3BAHrCQI62SycvO8W91K3DOrHM)

done 👍

**Comment by papjul** [comment]
Thank you for the details!

First of all, thank you for taking the time to design a banner for a project you like :)

I see many advantages to using raw screenshots:
- I can easily swap them if I want to make newer screenshots
- We can see many screenshots at once. In your suggestion, there is one single screenshot with most of the space being "empty", which doesn't help the user knowing all the features the app has to share
- We can see all the details, with no loss of space. On your suggestion, the screenshot is really small, we don't see much how the app looks
- It's responsive (on mobile, screenshots go one below the other, while with just one large image, the screenshot in the middle become even smaller). I don't know any way to make responsive images so it looks like a dead end for this item?

It looks to me like you could fix a few of these items, but unfortunately not all of them, what do you think?

Side note: "Best Weather App" is suggestive and I don't claim this title ;)

**Reply by KrishnaSSH** [reply]
## ☝️🤓I have suggestions to fix those hindrance
>I can easily swap them if I want to make newer screenshots
>
It just takes a little time to update a mockup so not an issue ig ;)

>We can see many screenshots at once. In your suggestion, there is one single screenshot with most of the space being "empty", which 
>doesn't help the user knowing all the features the app has to share
>We can see all the details, with no loss of space. On your suggestion, the screenshot is really small, we don't see much how the app looks
>
Yeah mb I just made a mocked up banner, if screenshots are the priority then their mockups can be created saperately like [this repo](https://github.com/DP-Hridayan/aShellYou) and [this repo](https://github.com/T8RIN/ImageToolbox) 

>It's responsive (on mobile, screenshots go one below the other, while with just one large image, the screenshot in the middle become even smaller). I don't know any way to make responsive images so it looks like a dead end for this item?
>
Mockups created saperately for each feature can be opted :)
